homeassistant/components/fyta/sensor.py

```diff
@@ -46,35 +46,35 @@ class FytaSensorEntityDescription(SensorEntityDescription):
         translation_key="plant_status",
         device_class=SensorDeviceClass.ENUM,
         options=PLANT_STATUS_LIST,
-        value_fn=lambda value: PLANT_STATUS[value],
+        value_fn=lambda value: PLANT_STATUS.get(value),  # pylint: disable=unnecessary-lambda
```

Comment:
```suggestion
        value_fn=PLANT_STATUS.get,
```
This would make pylint happy